x-pack/plugins/security_solution/scripts/endpoint/resolver_generator.ts

```diff
@@ -158,57 +186,17 @@ async function main() {
       default: false,
     },
   }).argv;
-  const pipelineName = 'endpoint-event-pipeline';
-  eventMapping.settings.index.default_pipeline = pipelineName;
+  await doIngestSetup(argv.kibana);
+
   const clientOptions: ClientOptions = {
     node: argv.node,
   };
-  if (argv.auth) {
-    const [username, password]: string[] = argv.auth.split(':', 2);
-    clientOptions.auth = { username, password };
-  }
+
   const client = new Client(clientOptions);
   if (argv.delete) {
-    await deleteIndices(
-      [argv.eventIndex, argv.metadataIndex, argv.alertIndex, argv.policyIndex],
-      client
-    );
+    await deleteIndices([argv.eventIndex, argv.metadataIndex, argv.policyIndex], client);
   }
 
-  const pipeline = {
-    description: 'redirects alerts to their own index',
-    processors: [
-      {
-        set: {
-          field: '_index',
-          value: argv.alertIndex,
-          if: "ctx.event.kind == 'alert'",
-        },
-      },
-      {
-        set: {
-          field: 'mutable_state.triage_status',
-          value: 'open',
-        },
-      },
-    ],
-  };
-  try {
-    await client.ingest.putPipeline({
-      id: pipelineName,
-      body: pipeline,
-    });
-  } catch (err) {
-    // eslint-disable-next-line no-console
-    console.log(err);
-    // eslint-disable-next-line no-process-exit
-    process.exit(1);
-  }
-
-  await createIndex(client, argv.alertIndex, alertMapping);
-  await createIndex(client, argv.eventIndex, eventMapping);
-  await createIndex(client, argv.policyIndex, policyMapping);
-  await createIndex(client, argv.metadataIndex, metadataMapping);
   if (argv.setupOnly) {
```

Comment:
Does `setupOnly` have value as an option anymore? I think if someone **only** wants to setup the indices and they have kibana running already they should just login to kibana, right?

Reply:
That's correct they could just log into kibana to get the package installed. 🤷‍♂️  I can remove it if you like.

Reply:
Yeah let's remove it